kong/plugins/proxy-cache/clustering/compat/response_headers_translation.lua

```diff
@@ -0,0 +1,12 @@
+local function adapter(config_to_update)
+    if config_to_update.response_headers["Age"] ~= nil then
+        config_to_update.response_headers.age = config_to_update.response_headers["Age"]
```

Comment:
This currently only sets the value of the `Age` field to the `age` field. We'd also have to remove the `Age` field from the adapted configuration.

```suggestion
        config_to_update.response_headers.age = config_to_update.response_headers["Age"]
        config_to_update.response_headers["Age"] = nil
```

Reply:
removed `Age` field. thanks